docs/tutorials/IndexerWriter.md

```diff
@@ -0,0 +1,132 @@
+# Conduit Indexer Writer
+
+This document explains how to configure and deploy Conduit to populate a
+PostgreSLQ DB suitable to serve as a backend for the [Indexer](indexer-readme) API.  This is
+not a production deployment. Treat this tutorial as an example configuration
+or a development environment. Each of these components have their own
+requirements for managing deployments, upgrades, scaling, and high
+availability.
+
+The components you will configure and deploy are:
+* an algod node configured with follow mode.
+* a PostgreSQL database.
+* a Conduit pipeline configured to pull data from algod and push it to the PostgreSQL database.
+
+```mermaid
+graph LR;
+    algod["algod"]
+    pe["postgresql Exporter"]
+    algodimp["algod Importer"]
+    db["PostgreSQL DB"]
+    indexer["Indexer REST API Service"]
+    algod-->algodimp
+    subgraph "Conduit Pipeline"
+        algodimp-->pe;
+    end
+    pe-->db;
+    db-->indexer
+```
+
+## Pre-requisites
+
+* Linux or Mac Operating System. At this time Windows is not supported by algod, so the bare metal installation described below will not work.
+* Docker is used for the database, so the `docker` command is required.
+
+## Algod
+
+The blockchain data source. Algod is controlled by Conduit to ensure the two
+services are always operating on the same block. In this way algod is able to
+store a minimal subset of blockchain data required for streaming data through
+Conduit.
+
+We will cover a basic installation using `update.sh`, for [additional
+information refer to the developer portal.](node-install-doc)
+
+```bash
+# download update.sh
+wget https://raw.githubusercontent.com/algorand/go-algorand/rel/stable/cmd/updater/update.sh
+
+# Define which network to run on. This selects the correct genesis file. 
+# Options: mainnet, testnet, betanet, devnet
+NETWORK=mainnet
+
+# download algod binaries and initialize data directory.
+update.sh -n -i -c stable -p bin -d algod_data -g $NETWORK
+
+# configure node to work with conduit
+./bin/algocfg profile set conduit -d algod_data
+```
+
+At this stage you have all of the algod tools in `bin` and an `algod_data`
+directory configured for Conduit with the desired network.
+
+Start the node:
+```bash
+~$ ./bin/goal node start -d algod_data
```

Comment:
```suggestion
./bin/goal node start -d algod_data
```
to stay consistent with the above?

Reply:
Thanks. This is the format I usually use, but for some reason above I wrote it as though you would run a script.